DBus isn't used by this so don't bother checking for it.

https://bugs.launchpad.net/ubuntu/+source/pam-xdg-support/+bug/1192600

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>

=== modified file 'Makefile.am'
--- Makefile.am	2012-09-27 06:53:57 +0000
+++ Makefile.am	2013-06-19 15:05:17 +0000
@@ -1,6 +1,3 @@
-AM_CPPFLAGS = 					\
-	$(LIBDBUS_CFLAGS)
-
 pammoddir = /lib/security
 pammod_LTLIBRARIES = 				\
 	pam_xdg_support.la
@@ -8,8 +5,7 @@
 pam_xdg_support_la_SOURCES = pam_xdg_support.c
 pam_xdg_support_la_LDFLAGS = -no-undefined -module -avoid-version
 pam_xdg_support_la_LIBADD =  			\
-	$(PAM_LIBS)				\
-	$(LIBDBUS_LIBS)
+	$(PAM_LIBS)
 
 man_MANS = pam_xdg_support.8
 

=== modified file 'configure.ac'
--- configure.ac	2012-10-02 03:09:28 +0000
+++ configure.ac	2013-06-19 15:05:17 +0000
@@ -22,8 +22,6 @@
 AC_CHECK_HEADERS(paths.h)
 AC_CHECK_HEADERS(sys/consio.h)
 
-PKG_CHECK_MODULES(LIBDBUS, dbus-1)
-
 dnl ---------------------------------------------------------------------------
 dnl Check for PAM
 dnl ---------------------------------------------------------------------------

